src/main/js/AutoCloseWordMapping.tsx

```diff
@@ -37,6 +37,12 @@ type MappingProps = {
   update: (mapping: Mapping) => void;
 };
 
+const ShrinkedInputField = styled(InputField)`
+  &.field.is-grouped > .control {
```

Comment:
This should be fixed in the InputField component. Please do not use styled components to access child selectors across component boundaries.